src/main/java/org/jabref/gui/bibtexextractor/BibtexExtractorViewModel.java

```diff
@@ -58,6 +64,16 @@ public StringProperty inputTextProperty() {
     public void startParsing() {
         BackgroundTask.wrap(() -> currentCitationfetcher.performSearch(inputTextProperty.getValue()))
                       .onRunning(() -> dialogService.notify(Localization.lang("Your text is being parsed...")))
+                      .onFailure((e) -> {
+                          if (e instanceof UncheckedIOException) {
+                              String msg = Localization.lang("There are connection issues with the %0 server. Detailed information: %1.",
```

Comment:
I think it's not so important for the user which exact url failed (especially since the popup disappears to quickly to type it in the url of a webbrower to check). So I suggest the following wording: "Could not connect to JabRef server (Connection timed out)."

Reply:
And maybe add the details via error log.

Reply:
Fair point, @tobiasdiez.
This should be "Could not connect to GROBID server (Connection timed out)." Shouldn't it?

Reply:
I would still say "JabRef server"...the user knows what Jabref is, but not necessarily what grobid is. I would see this also as an "implementation detail", maybe in the future we switch the backend from grobid to something else..

Reply:
It was tricky to get the Message to a phrase as you proposed.
I reduced the level of detail towards:
"There are connection issues with a JabRef server. Detailed information: %0."
(See latest screenshot)

Is this OK for you @tobiasdiez ?

Reply:
Perfect! Thanks